pytest.ini

```
@@ -34,3 +34,5 @@ markers =
     no_new_console: Prevent creating a new IPython console when reusing a mainwindow instance
     close_main_window: Close main window instance after test
     no_web_widgets
+
+addopts = --ignore=./external-deps
```

```suggestion
addopts = --ignore=./external-deps

```

Add an empty line here, which we tend to do for all our files.